docs/siem/index.asciidoc

```diff
@@ -1,60 +1,161 @@
+[chapter]
 [role="xpack"]
 [[xpack-siem]]
-= Elastic Security
-
-[partintro]
---
+= Elastic Security overview
```

Comment:
At one point I think the titles in the nav list were meant to align with the app names in Kibana. If that's still true, you might want to consider using a "titleabbrev" (like in https://raw.githubusercontent.com/elastic/kibana/master/docs/user/introduction.asciidoc) to show the shorter name in the nav.